packages/headless/doc-parser/src/entity.ts

```diff
@@ -1,4 +1,4 @@
-type EntityKind =
+export type EntityKind =
```

Comment:
Following on the comment below, I _think_ this type could become `private` again,